index.js

```diff
@@ -0,0 +1,80 @@
+// 1
+
+const movies = ["The Shawshank Redemption",
+    "The Godfather",
+    "The Dark Knight",
+    "12 Angry Men",
+    "The Lord Of The Rings",
+    "Pulp Fiction",
+    "The Good, the Bad and the Ugly",
+    "Forrest Gump"];
+
+    for (let index = 0; index < movies.length; index++) {
+        const element = movies[index];
+        console.log(element);
+       console.log(getWordCount(element))
+    }
+
+function getWordCount(str) {
+        
+     return str.trim().split(/\s+/).length;
+}
+
+// 2
+
+const body = document.querySelector('body');
+const btn = document.querySelector('.js-btn');
+
+body.insertAdjacentHTML('beforeend', `<button class="js-btn">Click</button>`);
+
+
+window.addEventListener('click', onClick);
```

Comment:
You have set the event listener on the `window` object, not your button. 😓 You wrote a correct event handler function though, as in `onClick` is a good function, you just should have put it on the button, not the window. This way, wherever you click on the page makes the event happen, and the window obviously never gets disabled. But you did have the correct idea though, so do not beat yourself up! 🎖️

Reply:
I tried to add the eventsLisner on the button, but I got an error. Now I understood why. I declare variable 'btn' before creating it. I didn't have time, so I found the solution with window.